src/comps/orth.cc

```diff
@@ -1,10 +1,11 @@
+#include <RandLAPACK.hh>
+#include <RandBLAS.hh>
```

Comment:
Two comments:

1. I think ``#include <RandLAPACK.hh>`` should be ``#include "RandLAPACK.hh"``.
2. It seems excessive to include all of RandLAPACK for this file. There is a good chance that ``#include "RandLAPACK/comps/orth.hh"`` would suffice.

Reply:
agree on both points. I'll see if your suggestion works.

Reply:
done